mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -9,19 +9,28 @@
     <CBehaviorBuff id="AdeptPhaseShift">
         <Modification MoveSpeedBonus="0"/>
     </CBehaviorBuff>
+    <CBehaviorBuff id="AdeptPhaseShiftCaster">
+        <Modification>
+            <StateFlags index="SuppressPassenger" value="0"/>
+        </Modification>
+    </CBehaviorBuff>
     <CBehaviorBuff id="AdeptPhaseShiftTimer">
         <RemoveValidatorArray value="CasterIsNotRecallingNexus"/>
+        <RemoveValidatorArray value="CasterIsNotHidden"/>
     </CBehaviorBuff>
     <CBehaviorBuff id="BatteryEnergy">
         <InfoFlags index="Hidden" value="1"/>
         <EditorCategories value="AbilityorEffectType:Units"/>
         <Duration value="0.01"/>
         <ExpireEffect value="BatteryAddEnergy"/>
     </CBehaviorBuff>
+    <CBehaviorBuff id="BroodlingFate">
+        <Duration value="5"/>
+    </CBehaviorBuff>
     <CBehaviorBuff id="DarkTemplarBlinkAttackDelay">
         <InfoFlags index="Hidden" value="1"/>
         <EditorCategories value="Race:Protoss,AbilityorEffectType:Units"/>
-        <Duration value="1.05"/>
+        <Duration value="1"/>
```

Comment:
Dark Templar's Blink next attack delay reduced from 0.75 to 0.71 seconds (from 1.05 to 1 on normal game speed)

The effective delay's value should have been 1.0625 since 1.05 is not the end of a game update